modules/ROOT/pages/deployment-status-messages.adoc

```diff
@@ -0,0 +1,131 @@
+Permanent warning messages status usually requires your action to be fixed. Instead, temporary warning messages disappear by themselves. Though, it is advised that you check them with your Kubernetes admin if they don't disappear in reasonable time.
+
+For example, if a node goes down, you likely have pods pending temporarily. Your Kubernetes admin may need to bring that node up manually. Or maybe the machine is taking its time to be rebooted.
+
+A Mule app deployment may return a CrashLoopBackOff warning because the Mule app fails to start due to an external dependency being down, such as a database or JMS server. The warning may fix itself when the service comes up. Though, if there is something wrong with the deployment artifact, then the warning message will be permanent.
+
+== Warning Messages Description
+
+The following table lists different types of warning messages. Note that this is not an exhaustive list and for the actual reference you must refer to the Kubernetes documentation.
+
+[%header%autowidth.spread]
+|===
+|Warning Status|Type |Description |Example
+|PodInitializing|Usually temporary |This status means that there is a pod initialization delay.  Which means that not all containers in your Mule app pods are ready and that it is usually just a matter of time until the pods initialize. If there is a real problem with your deployment, this warning transitions into another state. |<<link-example-1,Link>>
+|CrashLoopBackOff|Usually permanent |This status means there is a problem with your Mule app image and the app is crashing. |<<link-example-2,Link>>
+|Unschedulable|Usually temporary |This status means that there are no nodes available to run the requested replicas. There could be a few reasons, for example: 
+
+* Some nodes may be down for maintenance. +
+* Nodes may not have enough resources available to assign for your deployment. +
+* The *Enforce deploying replicas across nodes* option the *Deployments Target* tab of Runtime Manager is enabled but there are not enough different nodes available for the number of replicas desired. |<<link-example-3,Link>>
+|===
+
+For more information on Pod Status, refer to the https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-lifetime[Kubernetes official documentation^]. Your Kubernetes administrator is also a great resource to inquire about whether a state should be considered critical or not and advise how to troubleshoot if required.
+
+== Warning Messages Examples
+
+[[link-example-1]]
+=== PodInitializing Example
+In the following example, the warning message means the pod is ready but is waiting to initialize:
+
+image::rtf-deployment-example-1.png[PodInitializing warning status in Runtime Manager UI]
+
+[[link-example-2]]
+=== CrashLoopBackOff Example
+
+In the following example, the warning message  means there is a problem with your Mule app image. Because a corrupt artifact was uploaded and caused the Mule app to crash during the deployment.
```

Comment:
```suggestion
In the following example, the warning message means there is a problem with your Mule app image. Because a corrupt artifact was uploaded and caused the Mule app to crash during the deployment.
```